Add network traffic whitelisting guide for Agentless mode (#26188)
DataDog/documentation: content/en/tests/troubleshooting/_index.md

1. Go to the [**Tests**][3] page for the language you're instrumenting and check that the testing framework you are using is supported in the **Compatibility** section.
2. Check if you see any test results in the [**Test Runs**][4] section. If you do see results there, but not in the [**Tests**][5] section, Git information is missing. See [Data appears in Test Runs but not Tests](#data-appears-in-test-runs-but-not-tests) to troubleshoot it.

> 3. If you are reporting the data through the Datadog Agent, make sure there is [network connectivity][15] from the host where your tests are running to the host and port where the Agent is running. Make sure you run your tests with the appropriate Agent hostname set in the `DD_AGENT_HOST` and the appropriate port in `DD_TRACE_AGENT_PORT` environment variables. You can activate [debug mode][6] in the tracer to check if it's able to connect to the Agent.

Reviewer: ```suggestion
3. If you are reporting the data through the Datadog Agent, make sure there is [network connectivity][15] from your test-running host to the Agent's host and port. Run your tests with the appropriate Agent hostname set in the `DD_AGENT_HOST` and the appropriate port in `DD_TRACE_AGENT_PORT` environment variables. You can activate [debug mode][6] in the tracer to verify connectivity to the Agent.
```

> 4. If you are reporting the data directly to Datadog ("Agentless mode"), make sure there is [network connectivity][16] from the host where your tests are running to Datadog's hosts. You can activate [debug mode][6] in the tracer to check if it's able to connect to Datadog.

Reviewer: ```suggestion
4. If you are reporting the data directly to Datadog ("Agentless mode"), make sure there is [network connectivity][16] from the test-running hosts to Datadog's hosts. You can activate [debug mode][6] in the tracer to verify connectivity to Datadog.
```
